health_check/contrib/rabbitmq/backends.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from kombu import Connection
+from amqp.exceptions import AccessRefused
+
+from django.conf import settings
+
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import ServiceUnavailable
+
+logger = logging.getLogger(__name__)
+
+
+class RabbitMQHealthCheck(BaseHealthCheckBackend):
+    """
+    Healthchecker for RabbitMQ.
+    """
+
+    def check_status(self):
+        """
+        Checks if RabbitMQ service is up by opening and closing
+        a broker channel.
+        """
+        logger.info("Checking for a broker_url on django settings...")
+
+        broker_url = getattr(settings, "BROKER_URL", None)
+
+        try:
+            logger.info("Got %s as the broker_url. Connecting to rabbit...")
+
+            conn = Connection(broker_url)
+
+            # opens and closes a channel to rabbitmq to make sure the service is up
+            logger.info("Openning a channel to RabbitMQ...")
+            conn.channel()
+
+            logger.info("Closing the channel to RabbitMQ...")
+            conn.close()
```

Comment:
is there a context manager you could use? Even if something fails the connection should be closed